src/transformers/models/maskformer/convert_maskformer_original_pytorch_checkpoint_to_pytorch.py

```diff
@@ -0,0 +1,728 @@
+# coding=utf-8
+# Copyright 2022 Facebook AI Research and The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# import sys
+# import sys
```

Comment:
```suggestion
import sys
```
This should be uncommented.